experiment/secret-sync-controller/SecretSyncs-config.yaml

```diff
@@ -0,0 +1,16 @@
+- source: 
```

Comment:
naming nit of the filename, prefer lowercase; in this case maybe just rename to `config.yaml`

Reply:
Filenames prefer lowercase with dashes?
Renamed.